csharp/ql/integration-tests/linux-only/compiler_args/global.json

```diff
@@ -0,0 +1,5 @@
+{
+    "sdk": {
+        "version": "8.0.100"
```

Comment:
Depending on the merge order, this PR is relevant: https://github.com/github/codeql/pull/15475